govcd/api_token.go

```diff
@@ -21,122 +21,213 @@ import (
 	"github.com/vmware/go-vcloud-director/v2/util"
 )
 
-// SetApiToken behaves similarly to SetToken, with the difference that it will
-// return full information about the bearer token, so that the caller can make decisions about token expiration
-func (vcdClient *VCDClient) SetApiToken(org, apiToken string) (*types.ApiTokenRefresh, error) {
-	tokenRefresh, err := vcdClient.GetBearerTokenFromApiToken(org, apiToken)
+// TODO Have distinct names for API and Refresh tokens
```

Comment:
is this TODO for now, or for future?

Reply:
For future, right now we can't change the public functions and as we have API token/Service Account overlapping RefreshToken structure, it would be nice to have a distinction in code between API tokens and Refresh tokens.